src/components/pages/profile/profile-form.tsx

```diff
@@ -40,7 +40,10 @@ const ProfileForm: FC<FormProfileProps> = ({ className }) => {
   return (
     <Form {...form}>
       <form
-        className={cn('space-y-8 pr-[9.75rem]', className)}
+        className={cn(
+          'space-y-8 xs:max-w-[343px] md:max-w-none md:pr-16 lg:pr-[9.75rem]',
```

Comment:
Here we also can remove `space-y-8` and use independent margins for children

```suggestion
          'xs:max-w-[343px] md:max-w-none md:pr-16 lg:pr-[9.75rem]',
```